### PR TITLE
Adding file size to File browser hover

### LINKED ISSUE #7352: Show file-size in File browser
Hi, 
I believe developers are frequently looking for the **size of the files** in File Browser.
One simply needs to to see how big are input *.csv files, *.pickle files or many other.
One still has to switch to the OS file-browser, to see this very basic information.

Would you find it useful to add and display this information in the FileBrowser?

_Note:
I could not find any plugin/extension, that could do this. 
But is it generally useful basic thing, that this feature should be standard part of JupyterLab interface._

### PR DESCRIPTION
**Pull request by macwoj**
## References

Fixes #7352

## Code changes

added size field to the Contents.IModel
added more file context to title in updateItemNode (filebrowser/src/listing.ts)

## User-facing changes

Hover in file vrowser will now show.

Name:
Size: If available the human readable file size with a single decimal.
Path: only the directory limited to 50
Created: YYYY-MM-DD HH:mm:ss
Modified: YYYY-MM-DD HH:mm:ss

## Backwards-incompatible changes

added new field which is optional so no backwards compatibility issues.

### CODE REVIEW

File: packages/filebrowser/src/listing.ts [hunk 1]
```diff
@@ -1833,8 +1833,13 @@ export namespace DirListing {
         // clean up the svg icon annotation, if any
         delete icon.dataset.icon;
       }
+      // add file size to pop up if its available
+      if (model.size !== null && model.size !== undefined) {
+        node.title = model.name + ' - ' + Private.formatFileSize(model.size, 1);
```
Comment (blink1073) [inline packages/filebrowser/src/listing.ts]: Thanks for adding this!  I think it looks too sparse without adding more info.  I think we should show the following:

Name: foo.bar
Size: as you have it
Path: path limited to 50 chars with an ellipsis if longer
Created:  Formatted using [Time.formatHuman](https://github.com/jupyterlab/jupyterlab/blob/1aff4190084fd5993d3f7e3ae10b467264df1cd0/packages/coreutils/src/time.ts#L17).
Modified: Same as above

Similar to what Finder shows on MacOS about a file:

image

Comment (blink1073) [reply packages/filebrowser/src/listing.ts]: And just not show the size part if we don't have it.

Comment (macwoj) [reply packages/filebrowser/src/listing.ts]: Yeah that looks better. Let me fix it.

**Comment by blink1073** [comment]
Perfect, thanks again @macwoj!

**Pull request merged**